.travis.yml

```diff
@@ -25,6 +25,11 @@ cache:
 
 before_script:
   - phpenv config-add .travis.php.ini
+  # TODO: cleanup after FGA implementation!
```

Comment:
Is there a chore in pivotal to clean this up afterwards? Do you now if it was possible to add `mkdir -p /src/dependencies/` to Travis?

Reply:
Yes, just created that [chore](https://www.pivotaltracker.com/story/show/161310060)

Reply:
And I've tried creating the /src/dependencies folder. But was not allowed to do so.